src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -140,6 +146,22 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     immutable.Map(googleKey -> servicesStart)
   }
 
+  protected def getMasterInstanceStartupScript(welderEnabled: Boolean): immutable.Map[String, String] = {
```

Comment:
Remove `masterInstanceStartupScript` because this is its replacement.

Reply:
+1

Reply:
I left it there because there was a discussion around removing the enableWelder field for cluster creation in the future when welder is released on both AoU and Terra, so we could go back to using `masterInstanceStartupScript`

Reply:
I would still remove it -- it's confusing to have duplicated code. If we need the old version we have git history. I would say leaving it as a `def` going forward sounds fine though.